scm/driver/gitlab/issue.go

```diff
@@ -291,7 +291,7 @@ type issueAssignee struct {
 
 type issueComment struct {
 	ID     int `json:"id"`
-	Number int `json:"noteable_iid"`
+	Number int `json:"notable_iid"`
```

Comment:
Is this a typo or is this what gitlab actually sends?

Reply:
Check the gitlab api documentation.

Reply:
Seems like it's noteable_iid, see: https://docs.gitlab.com/ee/api/issues.html#promote-an-issue-to-an-epic
revert the change, and use the nolint directive.